csharp-intermediate-topics/RenameFilesWithCSharp/RenameFilesWithCSharp/FileRenameService.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace RenameFilesWithCSharp;
+
+public static class FileRenameService
+{
+    public static void RenameFileWithFileClass(string oldFile, string newFile)
+    {
+        File.Move(oldFile, newFile);
+    }
+
+    public static void ExecuteRenameFileWithFileClass()
+    {
+        var oldFile = @"C:\MyDirectory\File1.txt";
+
+        var newFile = @"C:\MyDirectory\File_v1.txt";
```

Comment:
Same as in the previous service, let's remove this empty line between two variable declarations.